docs/cloud-workload-security/linux_expressions.md

```diff
@@ -3625,6 +3701,26 @@ Definition: Path argument of the syscall
 
 Constants are used to improve the readability of your rules. Some constants are common to all architectures, others are specific to some architectures.
 
+### `ABI` {#abi}
+ABI used for binary compilation.
+
+| Name | Architectures |
+| ---- |---------------|
+| `BIT32` | all |
```

Comment:
Isn't it redundant with `Architecture` ?

Reply:
I don't think, as we could have binaries using the 32bit ABI of 64bit architectures